oe_webtools_analytics/tests/src/Kernel/WebtoolsImportSettingsServiceTest.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\Tests\oe_webtools_analytics\Kernel;
+
+use Drupal\KernelTests\KernelTestBase;
+use Drupal\oe_webtools_analytics\EventSubscriber\WebtoolsImportSettingsSubscriber;
+
+/**
+ * Test to ensure 'WebtoolsImportSettingsSubscriber' service is reachable.
+ *
+ * @package Drupal\Tests\oe_webtools_analytics\Kernel
+ */
+class WebtoolsImportSettingsServiceTest extends KernelTestBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  public static $modules = ['oe_webtools_analytics'];
+
+  /**
+   * Test for existence of 'WebtoolsAnalyticsSearch' service.
+   */
+  public function testWebtoolsImportSettingsService() {
+    $subscriber = $this->container->get('oe_webtools_analytics.event_subscriber');
```

Comment:
If you want to kernel test this I would propose to make a test module that you enable during the test, that implements the event, and sets some values. Then you can assert the values.